torch/distributed/_tensor/debug/comm_mode.py

```diff
@@ -220,17 +234,33 @@ def add_json_information(json_dict, fqn):
             # adds module operation information
             forward_operations = []
             backward_operations = []
-
-            if fqn in self.comm_module_operation_counts:
+            if noise_level >= 2:
```

Comment:
Instead of conditioning on the single variable `noise_level`, would it make more sense to use `noise_level` to determine a set of flags which you can condition on here.

E.g. if `noise_level = 2`, we set `include_ops=True` and `include_trivial_ops=False`.

The advantage is that the code is more readable and maintainable. For example, if in the future we add more options, we can easily configure the `noise_level` to have more granular control.